src/components/dashboard/dashboard.tmpl.html

```diff
@@ -37,42 +37,42 @@
 
   <!-- Widgets selection container -->
   <div id="widget-selector" collapse="!showWidgetSelector()" ng-if="!customWidgetSelector.path">
-    <div class="title">
-      <p>Select the widgets you want to add to your dashboard.</p>
-      <span class="badge confirmation">Widget added!</span>
-      <i class="fa fa-times-circle" ng-if="showCloseWidgetSelectorButton()" ng-click="displayWidgetSelector(false)"/>
+    <div class="widget-selector__title">
```

Comment:
On this point, @cesar-tonnoir, and your message on slack "don't you think it will be a bit confusing for the developer if we have several conventions in the same lib?" I wanted to write down my thoughts on this and involve @alexnoox and start a little discussion, see what you guys think :)

In answer to your question, I feel like that’s kind of like saying refactoring some JS/Ruby code to follow a different convention/style from the rest of the project to improve maintainability & extensibility of that piece shouldn’t be done because it will confuse developers. 

I am just trying to experiment with some newer techniques, trying what people who put a lot of thought into making CSS easier to scale & more enjoyable to develop on are doing.

I do agree that applying a convention across a project is important, but I think applying new conventions to new / newly refactored components is a good way to make steps forward, making coming back to these components easier for our future selves.

So at this point you may be thinking what is the value of the BEM naming, and what is wrong with nesting in less/sass?

Most of our component classes are almost always nested 3+ levels deep, which creates some very specific classes. Specificity can, among other things:

- limit our ability to extend and manipulate stylesheets
- prevent things from working as expected when moved into different environments
- greatly reduce the modularity of our styles
- be very frustrating as a developer

My main goal with the way I want to be writing less/sass, is that I want my rules to match the elements I want, without them accidentally matching the elements I don’t, and where as the naming captures purpose and meaning that tells me more about what the piece of markup is doing.

So far trying out the BEM syntax I think one of the main advantages is that you have more idea of what the piece of markup is doing, and most importantly without the specificity. 

Originally when I looked at BEM I thought you had to represent each level on nesting with the double underscore. But let me show you an example:

What clicked for me is when I realised the 'block' is like defining the dependency for the 'element'. The `top-panel` element depends on the `widget` block, the `logo` element depends on the `top-panel`. So if I create another `widget` block, I know I can drop the `top-panel` and it's children classes in. 

```html
<div class="widget">
  <div class="widget__top-panel">
    <img class="top-panel__logo" />
    <p class="top-panel__text" />
  </div>
</div>
```

A bonus in being able to express a dependency on a modified block:

```html
<div class="section section--display-picture">
  ...
</div>
<div class="section section--about-me">
  <p class="section__body section--about-me__body" />
</div>
```

Once you experience making changes to classes like this, you realise that's its far less daunting to start moving things around, without the fear of breaking styles in unexpected locations. 

We don't have to follow the spec precisely, I think it's just about reducing Less/Sass nesting, and creating meaningful names that represent your design intentions.